malsim/__init__.py

```diff
@@ -31,8 +31,10 @@
 
 __all__ = ("AttackerEnv", "DefenderEnv", "register_envs")
 
+
+# TODO: Make sure logging dir exists and make it configurable (or use same as maltoolbox)
 formatter = logging.Formatter('%(asctime)s %(name)-12s %(levelname)-8s %(message)s', datefmt='%m-%d %H:%M')
-file_handler = logging.FileHandler('tmp/malsim_log.txt', mode='w')
+file_handler = logging.FileHandler('logs/malsim_log.txt', mode='w')
```

Comment:
Perhaps we could have a single config file for all these tools, not just one specific to the mal toolbox. The mal toolbox will still load it and the config will be importable from the toolbox to the simulator/gui/etc. It's just a conceptual thing I think and a matter of how we name the config file. Sth like `malconfig.yml` or `maltools.yml`.